.github/workflows/publish-operators-for-e2e-tests.yml

```
@@ -13,33 +16,42 @@
-    if: ${{ github.event.comment == '' || contains(github.event.comment.body, '/retest') || contains(github.event.comment.body, '/test all') || contains(github.event.comment.body, '/test e2e') }}
+    if: >-
+      ${{ github.event_name == 'pull_request_target' ||
+          (github.event_name == 'issue_comment' &&
+            (contains(github.event.comment.body, '/retest') ||
+             contains(github.event.comment.body, '/test all') ||
+             contains(github.event.comment.body, '/test e2e'))) }}
     steps:
-    # Checkout from PR event - in that case the comment field is empty
+    # Checkout from PR event
     - name: Checkout code from PR event
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 28: could not find expected ':'

(syntax-check)